package.json

```diff
@@ -7,5 +7,8 @@
     "test": "echo \"Error: no test specified\" && exit 1"
   },
   "author": "",
-  "license": "ISC"
+  "license": "ISC",
+  "dependencies": {
+    "typescript": "^3.9.7"
```

Comment:
You will also need to add a `tsconfig.json` file and a step to run `tsc` in order to compile the typescript.

Reply:
I was thinking to leave them in common JS.  Should I continue to give the example in  typescript?

Reply:
You can leave them in common JS - that's fine. In that case you can uninstall typescript.